mods/taskbar-clock-customization.wh.cpp

```diff
@@ -1179,15 +1186,48 @@ PCWSTR GetWeekdayFormattedWithExtra(std::vector<std::wstring>** extra) {
     if (g_weekdayFormatted.formatIndex != g_formatIndex) {
         const SYSTEMTIME* time = &g_formatTime;
 
-        auto weekdayFormatParts =
-            SplitTimeFormatString(g_settings.weekdayFormat.get());
-
-        GetDateFormatEx_Original(nullptr, DATE_AUTOLAYOUT, time,
+        auto weekdayFormatParts = SplitTimeFormatString(g_settings.weekdayFormat.get());
+
+        if (weekdayFormatParts[0] == L"custom") {
+            SYSTEMTIME lt; GetLocalTime(&lt);
+            if (g_settings.WeekdayFormatCustom.size() == 7) {
+                StringSetting M = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",0);
+                StringSetting T = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",1);
+                StringSetting W = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",2);
+                StringSetting R = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",3);
+                StringSetting F = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",4);
+                StringSetting S = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",5);
+                StringSetting U = Wh_GetStringSetting(L"WeekdayFormatCustom[%d]",6);
```

Comment:
Why load it again and not use `g_settings.WeekdayFormatCustom`?

Reply:
That's because I've got compiler errors before, but I see that if I use it directly in the copy, it seems to work

"error: call to implicitly-deleted copy constructor of 'StringSetting'
 StringSetting M = g_settings.WeekdayFormatCustom[0];"